yubihsm-container/Dockerfile

```diff
@@ -0,0 +1,44 @@
+# SPDX-FileCopyrightText: 2023 Technology Innovation Institute (TII)
+# SPDX-License-Identifier: Apache-2.0
+
+FROM ubuntu:jammy
+
+ENV YUBIHSM_PKCS11_CONF=/app/pkcs11.conf
+
+# Install dependencies
+RUN DEBIAN_FRONTEND=noninteractive apt-get update && apt-get install -y \
+    software-properties-common \
+    usbutils \
+    wget \
+    pkgconf libssl-dev opensc \
+    && rm -rf /var/lib/apt/lists/*
+    
+RUN apt-get update
+RUN apt-get install -y apt-utils
+RUN apt-get install -y libpcsclite1 libcurl4 libusb-1.0-0 libedit2                                                                                                                                                                    
```

Comment:
Should use frontend noninteractive for latter apt-get commands too.

Reply:
sure